app/assets/javascripts/discourse/config/deprecation-workflow.js

```diff
@@ -0,0 +1,6 @@
+self.deprecationWorkflow = self.deprecationWorkflow || {};
+self.deprecationWorkflow.config = {
```

Comment:
Super minor but I think we should probably stick to `globalThis` wherever possible? 😃 

```suggestion
globalThis.deprecationWorkflow = globalThis.deprecationWorkflow || {};
globalThis.deprecationWorkflow.config = {
```

Reply:
I think globalThis is correct. This came from ember-deprecation-workflow, so we should probably send a PR there.